tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -37,6 +37,8 @@
     no_log: true
     with_items:
       - "OVESETUP_CONFIG/adminPassword=str:{{ he_admin_password }}"
+      - "OVESETUP_DWH_DB/password=str:{{ ovirt_engine_setup_dwh_db_password | default() }}"
```

Comment:
It should be 'str:123456' or 'none:None' for an empty value 'str:' will be treated as an empty string.

Reply:
@tiraboschi I am almost sure I ran it without passing the argument, and it set the password to some random as expected.  But its been a while, and I am ok setting it to `123456` will update my PR.